templates/zerver/help/move-content-to-another-stream.md

```diff
@@ -47,8 +47,8 @@ destination streams.
 
 {!message-actions-menu.md!}
 
-1. Select **Move message** or **Edit message** (only one of these options will be shown). If you do
-   not see either option, you do not have permission to move this message.
+1. Select **Move message**. If you do not see this option, you do not have permission
```

Comment:
Message can also be moved using `Edit message` option but I guess this would be fine since the page is about moving message.

Reply:
Yeah, I think that's reasonable.